### PR TITLE
contrib_to_framework

### PR DESCRIPTION
**Pull request by sjay72**
New PR for Issue 8, section called "Contributing to the Framework."

Lucy, I had to re-mark-up my content. Please let me know if I bungled anything in this pass. If not, I believe that Andrew said that you or he would merge this PR.

### CODE REVIEW

File: doc-Community_Collaboration_Guide/topics/contributing_to_the_framework.adoc [hunk 1]
```diff
@@ -1,4 +1,152 @@
...
+origin	git@github.com:USERNAME/community-contribution-template.git (fetch)
+origin	git@github.com:USERNAME/community-contribution-template.git  (push)
+upstream	git@github.com:redhat-documentation/community-contribution-template.git (fetch)
+upstream	git@github.com:redhat-documentation/community-contribution-template.git (push)
+----
+
+[discrete]
+=== Fetch the Source and Create a Branch
+
+Fetch the latest source from the upstream repository. Note that using the `git fetch` command to obtain the latest copy of the source doesn not actually "put" the content on your local fork until you perform some other action, such as creating a branch.
```
Comment (lbopf) [inline doc-Community_Collaboration_Guide/topics/contributing_to_the_framework.adoc]: Just one small typo: doesn -> does

**Comment by lbopf** [comment]
Thanks, Susan!

Looks like all the changes got pulled over. I've just noted one typo (doesn -> does). If you can fix that up, we're good to merge.

(I also have some markup suggestions to get the code blocks to render as indented, but they're not at all urgent, I can submit those as an additional PR after this is all done.)

**Comment by sjay72** [comment]
@lbopf Typo fixed. Thanks for the re-review.

**Comment by lbopf** [comment]
Thanks, @sjay72! Merging this one now.

**Pull request merged**